.goreleaser.yaml

```diff
@@ -0,0 +1,47 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines below are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+version: 1
+
+before:
+  hooks:
```

Comment:
How/when are these Go releaser calls triggered?

Reply:
The before section allows for global hooks that will be executed before the release is started. So these are triggered when we run the command:
`goreleaser release`